clis/tcli/contrib/example_vpn_disable_linux.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+# This is a simple example script on how to use TCLID to turn off VPN and clean up routing.
+
+# NOTE: This is meant to be run from the libtelio directory.
+# And uses the debug build so that you could iterate quicker builds when debugging.
+
+TCLID_PATH="target/debug/tclid"
+
+# This script takes a single argument which is the name of the tunnel that Telio is using.
+if [ $# -eq 0 ]; then
+    echo "No tunnel name provided, using tun10 as default."
+    TUNNEL_NAME="tun10"
+else
+    TUNNEL_NAME=$1
+fi
+
+# Only Linux is supported at the moment.
+if [[ $(uname) != "Linux" ]]; then
+    echo "This script is only compatible with Linux."
+    exit 126
+fi
+
+# Check if the script is being run as root to precent half run scripts.
+if [ "$EUID" -ne 0 ]; then
+    echo "ERROR: This script must be run with root privileges."
+    exit 13
+fi
+
+if ! [ -f "$TCLID_PATH" ]; then
+    echo "Cannot find TCLID in $TCLID_PATH directory. Please build TCLID and run it from libtelio root."
+    exit 127
+fi
+
+# Check if the interface exists.
+if ! ip link show | grep -q "$TUNNEL_NAME"; then
+    echo "Tunnel $TUNNEL_NAME does not exists. Enable VPN before disabling it again."
+    exit 1
+fi
+
+# Ensure ip is installed.
+if ! command -v ip &> /dev/null; then
+    echo "iproute2 is required but not found. Please install it."
+    exit 127
+fi
+
+# Save current external IP to verif that it had changed later.
+old_public_ip=$(curl -s ifconfig.me)
+echo "Externa IP before enabling VPN: ${old_public_ip}"
+
+# Stop WireGuard.
+eval ${TCLID_PATH} dev stop
+# Stop libtelio daemon.
+eval ${TCLID_PATH} quit
+
+# Delete VPN route.
+# Routing into tunnel and routing from tunnel to VPN server will be deleted automatically when stopping Wireguard.
+ip rule del priority 32111 not from all fwmark 11673110 lookup 73110
```

Comment:
Same as: https://github.com/NordSecurity/libtelio/pull/450/files#r1606432734
```suggestion
ip rule del not from all fwmark 11673110 lookup 73110
```

Reply:
done